### ISSUE TITLE
Generator v0.10.0 fails trying to copy karma.conf.js

### ISSUE THREAD

**Issue by pdf** [Swiip/generator-gulp-angular#470]
Run the generator:
```bash
yo gulp-angular project-name
```

Select any options and after selecting a template engine get:
```
Template processing error on file karma.conf.js

events.js:72
        throw er; // Unhandled 'error' event
              ^
AssertionError: Trying to copy from a source that does not exist: /usr/lib/node_modules/generator-gulp-angular/app/templates/karma.conf.js
    at EditionInterface.exports._copySingle (/usr/lib/node_modules/generator-gulp-angular/node_modules/yeoman-generator/node_modules/mem-fs-editor/actions/copy.js:44:3)
    at EditionInterface.exports.copy (/usr/lib/node_modules/generator-gulp-angular/node_modules/yeoman-generator/node_modules/mem-fs-editor/actions/copy.js:22:17)
    at null.<anonymous> (/usr/lib/node_modules/generator-gulp-angular/app/src/write.js:39:19)
    at Array.forEach (native)
    at writeFiles (/usr/lib/node_modules/generator-gulp-angular/app/src/write.js:33:16)
    at /usr/lib/node_modules/generator-gulp-angular/node_modules/yeoman-generator/lib/base.js:409:16
    at processImmediate [as _immediateCallback] (timers.js:354:15)
```

**Comment by MCautreels** [comment]
I'm having the same problem. Previous version (v0.9.1) works perfectly fine.

**Comment by seanfcarroll** [comment]
I'm also having this issue.
http://stackoverflow.com/questions/29459131/cannot-install-gulp-angular

**Comment by andreierdoss** [comment]
+ 1 facing same issue

**Comment by jballands** [comment]
Same issue here

**Comment by cbentley** [comment]
I'm getting this, too.

**Comment by vinicius73** [comment]
+1

**Comment by bastienJS** [comment]
same issue!

![swiip](https://cloud.githubusercontent.com/assets/5945955/6998673/b6644b74-dbea-11e4-8fd1-b375af9f6ab7.PNG)

**Comment by usman8806** [comment]
I have a temp fix here:

Go to the directory where it says it can't find karma.conf.js, so in my case it said: 

**"AssertionError: Trying to copy from a source that does not exist: /usr/local/lib/node_modules/generator-gulp-angular/app/templates/karma.conf.js"**

I went to:
`/usr/local/lib/node_modules/generator-gulp-angular/app/templates/`

for me there was a **_karma.conf.js** in that folder, create a copy of that file and rename it to **karma.conf.js**.

After that I followed the generator prompts and everything worked!

**Comment by JustinLien** [comment]
This fixed it: https://github.com/wss-world/generator-gulp-angular/commit/8cb7ed679fb6dda1050bbbf68732e7384ffd82c5

For some reasons the update did not pull this change so I had to manually remove that line myself.

**Comment by anggiaj** [comment]
Still got the issue tho

**Comment by oliverjanik** [comment]
+1

**Comment by davguij** [comment]
@usman8806 Thanks, that fixes it.

**Comment by bastienJS** [comment]
@usman8806 still got the issue!

**Comment by sajjad-ser** [comment]
+1

**Comment by zckrs** [comment]
Not reproduce.

Can you provide some details ? 

node version, npm version, generator version, OS ?

**Comment by usman8806** [comment]
@bastienJS I saw that was a referenced fix up above but when I got that error I didn't have any files in the folder I ran the generator in. Where is this files.json located in?

**Comment by NeXTs** [comment]
Same issue here
following such properties: https://www.dropbox.com/s/s3us7tjeuf2fbal/%D0%A1%D0%BA%D1%80%D0%B8%D0%BD%D1%88%D0%BE%D1%82%202015-04-06%2016.46.52.png?dl=0

win7 x64. please ask if you need more details

**Comment by mistaanthony** [comment]
+1

**Comment by zckrs** [comment]
Fixed with [v0.11.0](https://github.com/Swiip/generator-gulp-angular/releases/tag/v0.11.0)

:warning: read changelog please

Thanks to reporting
